x-pack/plugins/cases/public/components/header_page/editable_title.tsx

```diff
@@ -71,6 +71,7 @@ const EditableTitleComponent: React.FC<EditableTitleProps> = ({
       onSubmit(newTitle);
     }
     setEditMode(false);
+    setErrors([]);
```

Comment:
Can you increase the width of the `EuiFieldText` so that all 64 chars are shown in edit mode

![image](https://user-images.githubusercontent.com/7405507/157428870-f229f461-6806-4c52-a672-f26c5f99cec7.png)

Reply:
Done.